### ISSUE TITLE
The tag generated by `Copy-ApiVersion.ps1` does not match [RCA] readme creation helper script error impact sdk automation 

### ISSUE THREAD

**Issue by jianingwang123** [Azure/azure-rest-api-specs#31973]
**Description:**

The tag generated in the `readme.md` file by the [Copy-ApiVersion.ps1](https://github.com/Azure/azure-rest-api-specs/blob/6067433c78a92ae5eb3b6b62ecc1511fe835320f/eng/scripts/Copy-ApiVersion.ps1) script does not match.

**Additional context:**

#31790 
https://github.com/Azure/azure-rest-api-specs/pull/31790#discussion_r1896068436
https://github.com/Azure/azure-sdk-tools/issues/9495

**Comment by wanlwanl** [comment]
@heaths @mikeharder could you take a look at this issue? This issues already impacted many sdk automations. Or @ the right person if I mention wrong people.

**Comment by heaths** [comment]
This was meant to be a helper script for people needing help with part of what the old hub did when creating new versions. It was not meant for automations.

**Comment by raych1** [comment]
I don't think this script is integrated into automations. The issue is the `readme.md` created by this script has incorrect tags, which causes the running failures in other automation tools.

**Comment by JackTn** [comment]
test pr: https://github.com/Azure/azure-rest-api-specs/pull/33694
@raych1  can you check and close this issue if ok !

**Comment by heaths** [comment]
> I don't think this script is integrated into automations. The issue is the `readme.md` created by this script has incorrect tags, which causes the running failures in other automation tools.

It tries to use standard conventions, but the user should make sure the output is correct (like with genAI responses). We tried to handle some corner cases, but there are so many in this repo it would require a major effort to support everything.
